dockerplugin.py

```diff
@@ -226,6 +240,53 @@ def read_memory_stats(container, dimensions, stats, t):
                    .format(_c(container)))
 
 
+def read_greedy_container_stats(container, dimensions, stats,
+                                container_inspect, sum_of_shares, t):
+    host_config = container_inspect['HostConfig']
+    cpu_quota = host_config.get('CpuQuota', 0)
+    cpu_percent = get_cpu_percent(stats)
+    num_cpus_host = len(stats['cpu_stats']['cpu_usage']['percpu_usage'])
+    # Get cpu quota used by container
+    if cpu_quota > 0:
+        if 'preread' in stats and 'precpu_stats' in stats:
+            period = host_config.get('CpuPeriod', 0)
+            # Default period length is 10000
+            cpu_period = 100000 if period == 0 else period
+            preread = datetime.strptime(
```

Comment:
`strptime` is pretty expensive. How does this scale when there are 1000+ containers on the same host? Is there a way to disable greedy container stats altogether if performance is an issue?

Reply:
I don't have an option as of now. You think it would nice to include a boolean configuration option?